src/tcgwars/logic/impl/gen7/HiddenFates.groovy

```diff
@@ -0,0 +1,1201 @@
+	BILL_S_ANALYSIS_51 ("Bill's Analysis", 51, Rarity.RARE, [TRAINER, SUPPORTER]),
+	BLAINE_S_LAST_STAND_52 ("Blaine's Last Stand", 52, Rarity.RARE, [TRAINER, SUPPORTER]),
+	BROCK_S_GRIT_53 ("Brock's Grit", 53, Rarity.UNCOMMON, [TRAINER, SUPPORTER]),
+	BROCK_S_PEWTER_CITY_GYM_54 ("Brock's Pewter City Gym", 54, Rarity.UNCOMMON, [TRAINER, STADIUM]),
+	BROCK_S_TRAINING_55 ("Brock's Training", 55, Rarity.HOLORARE, [TRAINER, SUPPORTER]),
+	ERIKA_S_HOSPITALITY_56 ("Erika's Hospitality", 56, Rarity.RARE, [TRAINER, SUPPORTER]),
+	GIOVANNI_S_EXILE_57 ("Giovanni's Exile", 57, Rarity.UNCOMMON, [TRAINER, SUPPORTER]),
+	JESSIE_JAMES_58 ("Jessie & James", 58, Rarity.HOLORARE, [TRAINER, SUPPORTER]),
+	KOGA_S_TRAP_59 ("Koga's Trap", 59, Rarity.UNCOMMON, [TRAINER, SUPPORTER]),
+	LT_SURGE_S_STRATEGY_60 ("Lt. Surge's Strategy", 60, Rarity.UNCOMMON, [TRAINER, SUPPORTER]),
+	MISTY_S_CERULEAN_CITY_GYM_61 ("Misty's Cerulean City Gym", 61, Rarity.UNCOMMON, [TRAINER, STADIUM]),
+	MISTY_S_DETERMINATION_62 ("Misty's Determination", 62, Rarity.UNCOMMON, [TRAINER, SUPPORTER]),
+	MISTY_S_WATER_COMMAND_63 ("Misty's Water Command", 63, Rarity.HOLORARE, [TRAINER, SUPPORTER]),
+	POKEMON_CENTER_LADY_64 ("Pokémon Center Lady", 64, Rarity.UNCOMMON, [TRAINER, SUPPORTER]),
+	SABRINA_S_SUGGESTION_65 ("Sabrina's Suggestion", 65, Rarity.UNCOMMON, [TRAINER, SUPPORTER]),
+	MOLTRES_ZAPDOS_ARTICUNO_GX_66 ("Moltres & Zapdos & Articuno-GX", 66, Rarity.ULTRARARE, [POKEMON, BASIC, POKEMON_GX, TAG_TEAM, _COLORLESS_]),
+	GIOVANNI_S_EXILE_67 ("Giovanni's Exile", 67, Rarity.ULTRARARE, [TRAINER, SUPPORTER]),
+	JESSIE_JAMES_68 ("Jessie & James", 68, Rarity.ULTRARARE, [TRAINER, SUPPORTER]),
+	MOLTRES_ZAPDOS_ARTICUNO_GX_69 ("Moltres & Zapdos & Articuno-GX", 69, Rarity.SECRET, [POKEMON, BASIC, POKEMON_GX, TAG_TEAM, _COLORLESS_]),
+;
+	
+	static Type C = COLORLESS, R = FIRE, F = FIGHTING, G = GRASS, W = WATER, P = PSYCHIC, L = LIGHTNING, M = METAL, D = DARKNESS, Y = FAIRY, N = DRAGON;
+	
+	protected CardTypeSet cardTypes;
+	protected String name;
+	protected Rarity rarity;
+	protected int collectionLineNo;
+
+	HiddenFates(String name, int collectionLineNo, Rarity rarity, List<CardType> cardTypes) {
+		this.cardTypes = new CardTypeSet(cardTypes as CardType[]);
+		this.name = name;
+		this.rarity = rarity;
+		this.collectionLineNo = collectionLineNo;
+	}
+
+	@Override
+	public CardTypeSet getCardTypes() {
+		return cardTypes;
+	}
+
+	@Override
+	public String getName() {
+		return name;
+	}
+
+	@Override
+	public Rarity getRarity() {
+		return rarity;
+	}
+
+	@Override
+	public int getCollectionLineNo() {
+		return collectionLineNo;
+	}
+
+	@Override
+	public tcgwars.logic.card.Collection getCollection() {
+		return tcgwars.logic.card.Collection.HIDDEN_FATES;
+	}
+
+	@Override
+	public String toString() {
+		return String.format("%s:%s", this.name(), this.getCollection().name());
+	}
+
+	@Override
+	public String getEnumName() {
+		return name();
+	}
+
+	@Override
+	public Card getImplementation() {
```

Comment:
Reindent with 2 spaces, since project has editorconfig, use IDE functionality.

Reply:
Can you double check that it's been re-indented?  I'm not sure if I used the editorconfig extension correctly